create_publication.py

```diff
@@ -12,65 +12,55 @@
 import logging
 import timeit 
 from pathlib import Path
-from src.utils.data_connections import read_sql_file, get_df_from_server, make_database_connection
+from pyspark.sql import functions as F
+import requests
+
 from src.utils.file_paths import get_config
 from src.utils.logging_config import configure_logging 
-from src.processing.clean import calculate_years, process_columns
-from src.processing.derive_fields import gp_count_by_region, calculate_mean_years
+from src.utils.spark import create_spark_session 
+from src.data_ingestion.get_data import download_zip_from_url
+from src.data_ingestion.reading_data import load_csv_into_spark_data_frame
+from src.processing.aggregate_counts import get_aggregate_counts, get_grouped_aggregate_counts
+from src.data_exports.write_csv import save_spark_dataframe_as_csv, rename_csv_output
+
 
 logger = logging.getLogger(__name__)
 
 def main():
     
     # load config, here we load our project's parameters from the config.toml file
     config = get_config("config.toml") 
```

Comment:
Default value is already "config.toml".

Reply:
It would be good to change the config default value to use a Path object from the Pathlib library (and generally change over to use Path objects; they are nice to work with
```suggestion
    config = get_config() 
```)